docs/usage/7-middleware/3-builtin-middlewares/2-allowed-hosts-middleware.md

```diff
@@ -1,15 +1,28 @@
 # Allowed Hosts
 
-Another common security mechanism is to require that each incoming request has a "HOST" header, and then to restrict
-hosts to a specific set of domains - what's called "allowed hosts". To enable this middleware simply pass a list of
-trusted hosts to the [Starlite constructor][starlite.app.Starlite]:
+Another common security mechanism is to require that each incoming request has a "Host" or "X-Fowarded-Host" header,
+and then to restrict hosts to a specific set of domains - what's called "allowed hosts".
+
+Starlite includes an `AllowedHostsMiddleware` class that can be easily enabled by either passing an instance of
+[TrustedHostsConfig][starlite.config.TrustedHostsConfig] or a list of domains to
```

Comment:
Shouldn't it be `starlite.config,AllowedHostsConfig`?

Reply:
fixed